### DISCUSSION TITLE
Week 7 is gonna be bonkers for desktop

### DISCUSSION THREAD

**Discussion by isakube** [FunkinCrew/Funkin#1743] [General]
The newgrounds version is JUST html5. With desktop you can do a lot more shit, and polymod is probably gonna get fixed up and in-game. desktop gonna be coolio, and it just might be the start of proper base-game modding. gaming.

**Comment by steviegt6** [comment]
The game would stay more or less identical for both release platforms, though? Polymod or some other modding framework could be stacked further on top of desktop builds, sure, but the base game would remain more or less identical.

**Reply by isakube** [reply]
yeah. i mean it pretty much has to be identical, minus desktop-only features. otherwise people get angy about lack of parity lol

fun fact: 954dd76ddab5c0e501138d5e1b6bdc2479740401 broke the polymod integration. why, ninjamuffin

**Reply by steviegt6** [reply]
Polymod hardly functioned before, regardless. I'm hoping for better support than a glorified XML file loader anyway....

**Reply by isakube** [reply]
> I'm hoping for better support than a glorified XML file loader anyway....

polymod supports hscript, so we might be able to hscript some important functions like the Note press event and graphic loading for the Note module. then we can do funny gimmicks like tiky notes

**Reply by isakube** [reply]
> Polymod hardly functioned before, regardless.

the reason is because polymod couldn't make up its mind as to which framework we were using. we used Lime's TEXT grabber, but used OpenFL for the rest (pretty sure.) That, and I think Flixel has its own asset grabbers as well, and Polymod doesnt support Flixel. So we'd have to change all our asset grabbers to be from one framework (probably Lime.) Then we'd have to force the framework to work with Lime and not some other framework like OpenFL. Lot of work.
